### DISCUSSION TITLE
Visibility of IP Prefix

### DISCUSSION THREAD

**Discussion by f0r7y** [netbox-community/netbox#10226] [Q&A]
Hi
I don't know if I understand the netbox organization correctly. I would like to know if it is possible to limit the visibility of a prefix to some structure (tenant, site, location, etc.). I created 2 prefixes associated with 2 different tenants but when I go to assign an ip I see all the devices registered in all tenants. it's correct?

**Comment by richbibby-NS1** [comment]
Hi, 
Not sure if this is exactly what you are looking for, but have you tried creating a filter in the Web UI? For example from within prefixes: https://demo.netbox.dev/ipam/prefixes/
click the 'filters' tab (next to results) and build the search based on a particular tenant.
